mods/voidmulti.sc2mod/base.sc2data/gamedata/upgradedata.xml

```diff
@@ -548,11 +548,9 @@
         <ScoreAmount value="200"/>
         <ScoreResult value="BuildOrder"/>
         <!--EffectArray Reference="Effect,CycloneAirWeaponDamage,Amount" Value="6"/-->
-        <EffectArray Reference="Effect,CycloneAirWeaponDamage,AttributeBonus[Armored]" Value="20"/>
+        <EffectArray Reference="Effect,CycloneAirWeaponDamage,Amount" Value="10"/>
         <!--EffectArray Reference="Effect,CycloneWeaponDamage,Amount" Value="5"/-->
-        <EffectArray Reference="Effect,CycloneWeaponDamage,AttributeBonus[Armored]" Value="20"/>
-        <EffectArray Operation="Set" Reference="Button,LockOn,AlertTooltip" Value="Button/Tooltip/LockOnUpgraded"/>
-        <EffectArray Operation="Set" Reference="Button,LockOn,Tooltip" Value="Button/Tooltip/LockOnUpgraded"/>
+        <EffectArray Reference="Effect,CycloneWeaponDamage,Amount" Value="10"/>
```

Comment:
Mag-Field Accelerator damage bonus changed from +20 vs armored to +10 vs all

The tooltips were changing the text to mention bonus vs armored. Thus, they were removed